components/create_comment/create_comment.jsx

```diff
@@ -282,6 +293,8 @@ class CreateComment extends React.PureComponent {
         this.focusTextbox();
         document.addEventListener('paste', this.pasteHandler);
         document.addEventListener('keydown', this.focusTextboxIfNecessary);
+        document.addEventListener('keyup', this.setIsShiftPressed);
```

Comment:
Detecting the shift key pressed or not to allow "not formatted paste"

Reply:
I think the standard for mac is different: Option-Shift-Command-V

also, if I press and release shift (or any combination with shift) and I right-click->paste, would it count as shift+paste?

Reply:
This handle the keydown and the keyup, so is going to work fine meanwhile you are not pasting and at the same time executing a combination of keystrokes that involves shift (That would be pretty hard to do at the same time).

I going to add the mac support for this.